frappe/integrations/offsite_backup_utils.py

```diff
@@ -66,8 +66,8 @@ def get_latest_backup_file(with_files=False):
 
 
 def get_file_size(file_path, unit):
-	if not unit:
```

Comment:
This validation prevents sending '' for unit
Let's not remove it

Reply:
Hi @hasnain2808

I actually forget to remove that conditional since I had concluded that a better path will be to not bother finding the file size of a file whose path is `None` or empty string.

Reply:
@tundebabzy 
how about https://github.com/frappe/frappe/pull/15552/files#r788581217

Reply:
We need a default value it will fail [here](https://github.com/frappe/frappe/pull/15552/files#diff-d6b9a7d94568ce928c5d605d5d1dd25c5b5cddbf6bc486a471bae00fb039d1b7L76) otherwise
I think this is a better way
https://github.com/frappe/frappe/pull/15552/files#r788581217